src/App.jsx

```diff
@@ -1,5 +1,48 @@
+import React, { useEffect, useState } from 'react'; // нужно ли импортировать в каждый файл?
+import { getToken, searchLocation } from './api';
+import CurrentWeather from './components/CurrentWeather';
+import SearchLocation from './components/SearchLocation';
+import { url, authData } from './constants';
+
 function App() {
-  return <div>Hello world!</div>;
+  const [token, setToken] = useState('');
+  const [coords, setCoords] = useState({});
+  const [searchedLocation, setSearchedLocation] = useState({});
+
+  navigator.geolocation.getCurrentPosition(
```

Comment:
do you want to call the function on each render?

Reply:
no, refactored this moment